magicctapipe/io/gadf.py

```diff
@@ -29,6 +29,55 @@
 MJDREF = Time(0, format="unix", scale="utc")
 
 
+def telescope_combinations(config):
```

Comment:
This function is defined also in `io/io.py`. Are they different? if not, define it only once (I think it fits better in `io/io.py`) and then import it correspondingly where needed

Reply:
Indeed, they are the same function. I tried to define it only in the `io.py`, but when I call it in `gadf.py` it says that I cannot import a function that wasn't yet instantiated. Do you have any suggestions?

Reply:
well, you simply need to import it. Just add in `gadf.py`:

```python
from .io import telescope_combinations
```